scripts/generate_metadata.py

```diff
@@ -51,20 +52,20 @@
 }
 
 
-def find_folders_with_name() -> List[str]:
+def find_customs_folders() -> List[str]:
     matched_folders = []
     for dirpath, dirnames, _ in os.walk(ROOT_DIR):
         for dirname in dirnames:
             if CUSTOMS in dirname:
-                matched_folders.append(os.path.join(dirpath, dirname))
+                matched_folders.append(Path(dirpath) / dirname)
     return matched_folders
```

Comment:
Based on https://github.com/valory-xyz/mech-tools-dev/pull/13#discussion_r2192876016:

Is this the equivalent?

```python
return [str(p) for p in Path(ROOT_DIR).rglob("*") if p.is_dir() and CUSTOMS in p.name]
```

Moreover, I do not think we necessarily need to return `str` right? I think that the calling method is passing it to another method which can use a path object directly.